en/docs/get-started/quick-start-guide.md

```diff
@@ -460,35 +461,34 @@ A message appears to pick a scenario.
     
 3. Note down the `API key` and `secret` for later use.
 
-    ![create-client-id](../assets/img/tutorials/create-client-id.png)
+    ![create-client-id](../../../assets/img/get-started/create-client-id.png)
```

Comment:
this image is displaying the new console

Reply:
This is a Google console screenshot.